mappings/net/minecraft/client/realms/gui/screen/RealmsSlotOptionsScreen.mapping

```diff
@@ -26,6 +26,34 @@ CLASS net/minecraft/class_4423 net/minecraft/client/realms/gui/screen/RealmsSlot
 		ARG 4 activeSlot
 	METHOD method_21484 getSlotName ()Ljava/lang/String;
 	METHOD method_21486 saveSettings ()V
+	METHOD method_25255 (Lnet/minecraft/class_4185;)V
+		ARG 1 button
+	METHOD method_25256 (Lnet/minecraft/class_4185;)V
+		ARG 1 button
+	METHOD method_32495 (Lnet/minecraft/class_5676;Lnet/minecraft/class_5676;Lnet/minecraft/class_1267;)V
+		ARG 2 button
+		ARG 3 difficulty
+	METHOD method_32496 (Lnet/minecraft/class_5676;Lnet/minecraft/class_1934;)V
+		ARG 1 button
+		ARG 2 gameModeIndex
+	METHOD method_32497 (Lnet/minecraft/class_5676;Ljava/lang/Boolean;)V
+		ARG 1 button
+		ARG 2 commandBlocks
+	METHOD method_32500 (Lnet/minecraft/class_5676;Ljava/lang/Boolean;)V
+		ARG 1 button
+		ARG 2 forceGameMode
+	METHOD method_32502 (Lnet/minecraft/class_5676;Ljava/lang/Boolean;)V
+		ARG 1 button
+		ARG 2 spawnNPCs
```

Comment:
```suggestion
		ARG 2 spawnNpcs
```

This is currently inconsistent between `ServerPropertiesHandler` (`spawnNpcs`) and `RealmsSlotOptionsScreen` (`spawnNPCs`), so a change should be made later to fix that.

Reply:
The other instances of this issue have been fixed in pull request #2423.